examples/multi_label/classifier_chain.py

```diff
@@ -0,0 +1,86 @@
+"""
+============================
+Classifier Chain
+============================
+
+Demonstrate that a single classifier chain
+out performs 10 independent classifiers
```

Comment:
Also format so that it's about 79 char each line (hopefully your editor can do that for you)